### ISSUE TITLE
Multiple files

### ISSUE THREAD

**Issue by aartek** [OnwerkGmbH/node-versioneye-update#11]
Is it possible to run `versioneye-update` for multiple files at once? I.e `package.json` and `bower.json`? 
If possible, how can I do it?

**Comment by jdoose** [comment]
Currently this is not supported, although this is a great idea.

Currently your only option would be to make multiple calls to nodeversioneyeupdate. You could put this into a shell script/batch file.

**Comment by aartek** [comment]
Yes, I can do both, but if I'd make `versioneye-update --file package.json` and then `versioneye-update --file bower.json` the results of task for `bower.json` would override the results for `package.json`. Am I right?

**Comment by jdoose** [comment]
Ah, I see. You have to have two versioneye projects. A versioneye project is always only responsible for one file.
For two files you would create two versioneye projects. You can merge one project as a sub project into the other later on, if you want to.

For a typical web project containing a package.json and a bower.json file I perform usually the following steps:
* Create project for the "package.json" file, either by upload via the versioneye web page or by using the commandline parameter `--createproject`. If you are using the --createproject parameter rename the created ".versioneye-update.json" to ".package.versioneye-update.json"
*  Create project for the "bower.json", either by upload via the versioneye web page or by using the commandline parameter `--createproject`, If you are using the --createproject parameter rename the created ".versioneye-update.json" to ".bower.versioneye-update.json"
* If you did not use the --createproject parameter remember the project ids of the two versioneye projects
* on the VersionEye web page I merge the project for bower.json into the package.json project
This steps are for preparation.

During my jenkins job I will have a shell script that
* first uploads the package.json
  * either by command line parameters: `--file package.json --projectid <yourprojectidforpackagejson>`
  * or by config file `--configfile .package.versioneye-update.json`
* and after that uploads the bower.json
  * either by command line parameters: `--file bower.json --projectid <yourprojectidforbowerjson>`
  * or by config file `--configfile .bower.versioneye-update.json`

So if I understood your request correctly it is more a versioneye issue. But it still would be cool if node-versioneye-update would support more than one file.  I will give that a thought, thanks for pointing that out!

An I always thought it would be cool if the createproject parameter could create more than one versioneye projects (#13), that would make your use case even easier.

**Comment by aartek** [comment]
Thanks for this comprehensive answer!
